### PR TITLE
Individual DJ Page for DJs

### PR DESCRIPTION
**Pull request by binerys**
Now DJs can have their own page complete with a new option to add their bio in panel:
screen shot 2017-02-21 at 2 58 14 pm

I just implemented a basic layout, please let me know any improvements that can be made to the design :)

### CODE REVIEW

File: react/frontpage/components/DJPage.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    if (!dj) {
+      return(
+        <div>
+          <p> DJ page doesn't exist! </p>
+        </div>
+      );
+    }
+
+    return (
+      <div class="djPage">
```
Comment (ChrisLaganiere) [inline react/frontpage/components/DJPage.jsx]: `className` in React :)

File: react/frontpage/components/DJPage.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        <Row>
+          <Col xs={12} md={4}>
+            <RectImage src={ dj.picture || defaultDJPic } circle maxWidth="380px" />
+          </Col>
+          <Col xs={12} md={8}>
```
Comment (ChrisLaganiere) [inline react/frontpage/components/DJPage.jsx]: This is already inside of a bootstrap grid, and stacking them can be weird.. In this case it looks good at all sizes so this is 👍, just something to remember

**Comment by ChrisLaganiere** [comment]
Wow @binerys this is absolutely amazing!! 🏆 😊

Works perfectly and looks really good. I want everyone to see how it works, but once they do please merge this in ✅

**Comment by binerys** [comment]
Added the default DJ bio + the class name fix! Will merge in now

**Comment by ChrisLaganiere** [comment]
Pushed to uclaradio.com!

**Comment by hsuregan** [comment]
Looks great!

**Pull request merged**